docker/bookkeeper/entrypoint.sh

```diff
@@ -89,22 +89,6 @@ configure_bk() {
     fi
 }
 
-fix_bk_ipv6_check() {
-
-   # `${SCRIPTS_DIR}/common.sh` - a shell script in Bookeeper, runs this command: `/sbin/sysctl -n net.ipv6.bindv6only`.
-   # This command can return an error exit code `255` and error message:
-   #    `sysctl: cannot stat /proc/sys/net/ipv6/bindv6only: No such file or directory`
-   #
-   # If the `common.sh` file is sourced in a script that sets "exit on error" as true (set -e), the script shall return
-   # an error code `255`. Since, the `bookkeeper/bin/bookeeper` shell script sources the `common.sh` file, and it sets
-   # "exit on error" as true, we encounter this error when it is invoked. To avoid that error, here we modify the
-   # `common.sh` file to execute `/sbin/sysctl -n net.ipv6.bindv6only` only if file `/proc/sys/net/ipv6/bindv6only`
-   # is available.
-
-   # Replace the 22nd line with "if [ -f /sbin/sysctl ] && [ -f /proc/sys/net/ipv6/bindv6only ]; then".
-   sed -i "22s|.*|if [ -f /sbin/sysctl ] \&\& [ -f /proc/sys/net/ipv6/bindv6only ]; then|" ${BK_HOME}/bin/common.sh
-}
-
 initialize_cluster() {
```

Comment:
PR #4420 had added many of these functions from Apache/Bookkeeper which didn't work in stock Apache/Bookkeeper image at the time, so that they could be fixed here. The idea was that those fixes are made here, until the fixes are also made on Apache/Bookeeper side. 

Later, I had raised this PR https://github.com/apache/bookkeeper/pull/2219 with the same fixes we made here in Pravega and the PR got merged there, so some of these customized functions aren't needed anymore. As such we can return back to having pravega/bookkeeper entrypoint.sh invoking apache/bookkeeper entrypoint.sh to start bookies (see how this changed in the entrypoint.sh diff in https://github.com/pravega/pravega/pull/4420/files0. 

Now, whether we do that change here while upgrading to Bookkeeper `4.11.x` or do it separately in a different PR is an open question. I'd suggest doing that separately. If you agree, we should raise an issue to track that separate piece of work.

Reply:
Agree, let's keep changes of this upgrade at minimum, but track this cleanup so we can do that on a separate PR.